src/external_cli/wasm_bindgen.rs

```diff
@@ -27,3 +28,19 @@ pub(crate) fn bundle(bin_target: &BinTarget) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+/// Transforms the output from `wasm-bindgen --version` into a [Version].
+pub(crate) fn wasm_bindgen_cli_version(stdout: Vec<u8>) -> anyhow::Result<Version> {
+    let stdout = String::from_utf8(stdout)?;
```

Comment:
Nit: We can use `from_utf8_lossy` instead, which never fails and instead uses the replacement character if necessary.
Maybe a bit more robust to junky CLI output.

Reply:
nice catch! Thanks, that's way better!